kubemarine/procedures/install.py

```diff
@@ -511,6 +516,7 @@ def overview(cluster: KubernetesCluster) -> None:
         },
         "package_manager": {
             "configure": system_prepare_package_manager_configure,
+            "disable_unattended_upgrades": system_prepare_package_manager_disable_unattended_upgrades,
```

Comment:
Update tree of tasks for install and add_node in Installation.md and Maintenance.md?

Reply:
fixed